src/styles/atoms/_tab.scss

```diff
@@ -0,0 +1,83 @@
+@mixin dc-tab {
+    @include dc-cf;
+
+    display: block;
+    margin: 0;
+    padding: 0;
+    background-color: $dc-white;
+    list-style: none;
+    box-shadow: $dc-box-shadow100;
+}
+
+@mixin dc-tab--vertical {
+    display: flex;
+    flex-direction: column;
+}
+
+@mixin dc-tab__element {
+    display: block;
+    padding: $dc-space100;
+    float: left;
+    cursor: pointer;
+
+    &:hover {
+        color: $dc-orange40;
+    }
+}
+
+@mixin dc-tab__element--vertical(
+    $tab-active-selector: dc-tab__element--active
+) {
+
+    &.#{$tab-active-selector} {
+        @include dc-tab__element--vertical--active;
+    }
+}
+
+@mixin dc-tab__element--active {
+    color: $dc-orange40;
```

Comment:
@ashwinaggarwal Can we think about a way to have the color for tab customizable?

Something like:

```scss
// _variables.scss
$dc-tab-color: $dc-orange-40;
```

```scss
// _tabs.scss
// example for dc-tab__element--active mixin
@mixin dc-tab__element--active($color: $dc-tab-color) {
   color: $color;
}
// .... and so on for the other mixins?
```

@ashwinaggarwal WDYT?